prog/dftb+/lib_dftb/dftd4param.F90

```diff
@@ -12,6 +12,7 @@ module dftbp_dftd4param
   use dftbp_assert
   use dftbp_accuracy, only : dp
   use dftbp_constants, only : pi, AA__Bohr, symbolToNumber
+  use dftbp_encharges, only : eeqInput
```

Comment:
I was a bit confused about this part, since it is not enforced consistently. Fortunately, this [is going to change](https://github.com/bhourahine/dftbplus/tree/typeNameConvnt).